assets/scripts/segments/info.json

```diff
@@ -599,7 +599,89 @@
       "image": "bike-lane-02.jpg"
     },
     "details": {
-      "inbound|regular": {
+      "inbound|regular|sidewalk": {
+        "elevation": 1,
+        "minWidth": 5,
+        "graphics": {
+          "center": [
+            "bikes--biker-01-inbound",
+            "markings--straight-inbound-light"
+          ],
+          "repeat": "ground--asphalt"
```

Comment:
I'm not entirely sure where these details values are used, but I noticed that I had neglected to update this section. Wasn't sure if there was a better value for "repeat" for the sidewalk sections...

Reply:
This looks good. One of these files are outdated, and I think this is the one we're no longer using. However, I haven't double-checked to see why we haven't completely removed this file yet, so thank you for updating this and keeping it on par with the other segment configuration data.